### ISSUE TITLE
Incoming messages are not appearing for one channel and flowing in for other channels

### ISSUE THREAD

**Issue by pranavkulkarni7** [tdlib/td#2709]
I am listening to 3 channels. The messages for channel one are not flowing in since last two weeks. Has anyone faced this issue? Kindly help.

**Comment by levlam** [comment]
Messages in big chats aren't expected to be received in real time, but if the user has the chat opened, then they are expected to be received faster. In TDLib you can open a chat using `openChat`, but don't forget to call `closeChat` when the chat is closed. Usually, users have at most one chat opened. Also, ensure that you use up-to-date TDLib version.

**Comment by FordSmh** [comment]
Can confirm tdlib version was the problem, in my case anyway

**Comment by ne0c0de** [comment]
Yea, tdlib version was caused that. Suddenly I couldn't get messages on my server when I'm using tdlib. Only messages coming to server if I open the chat on real client (desktop or mobile)

Then I updated tdlib and started to get messages but after that I got messages with randomly delay starting between 2 seconds to more then a minute.

I suspect something changed on Telegram side but there's no clarification yet
